src/components/ScreenWrapper/index.tsx

```diff
@@ -7,6 +7,8 @@ type Props = IScrollViewProps & {
   children: ReactElement;
 };
 
+const GRAY_600 = '#202024';
```

Comment:
I think it would be better if we use `src/styles/theme.ts` constants instead.

Reply:
I created a file to store all colors constants